libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4860,35 +4860,41 @@ void GCS_MAVLINK::send_water_depth() const
     }
 
     RangeFinder *rangefinder = RangeFinder::get_singleton();
-    if (rangefinder == nullptr || !rangefinder->has_data_orient(ROTATION_PITCH_270)) {
```

Comment:
still need to check for `nullptr`, and like the other one you can early return is there is no pitch 270 orentation